src/System.Diagnostics.EventLog/ref/System.Diagnostics.EventLog.cs

```diff
@@ -147,6 +147,28 @@ public partial class EventSourceCreationData
         public string ParameterResourceFile { get { throw null; } set { } }
         public string Source { get { throw null; } set { } }
     }
+    public sealed partial class EventLogTraceListener : System.Diagnostics.TraceListener
+    {
+        public EventLogTraceListener() { }
+        [System.CLSCompliantAttribute(false)]
+        public EventLogTraceListener(System.Diagnostics.EventLog eventLog) { }
+        public EventLogTraceListener(string source) { }
+        [System.CLSCompliantAttribute(false)]
+        public System.Diagnostics.EventLog EventLog { get { throw null; } set { } }
+        public override string Name { get { throw null; } set { } }
+        public override void Close() { }
+        protected override void Dispose(bool disposing) { }
+        [System.Runtime.InteropServices.ComVisibleAttribute(false)]
```

Comment:
We seem to be not using ComVisible in general. I am not sure whether we should keep it or not.

Reply:
@ericstj thoughts on this one?